### PR TITLE
tidy up package.json and READMEs

### PR DESCRIPTION
**Pull request by bnewbold**
These are mostly tweaks to look better in npm. The identity and crypto packages got usage examples.

Will probably want to spend more time on some of these READMEs in the future, but want to push these small consistency tweaks.

### CODE REVIEW

File: packages/common-web/README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
-# ATP Common Library for Web
+# @atproto/common-web
 
-A library containing code which is shared between web-friendly ATP packages.
+Shared Typescript code for other `@atproto/*` packages, which is web-friendly (runs in-browser).
```
Comment (devinivy) [inline packages/common-web/README.md:3]:
```suggestion
Shared TypeScript code for other `@atproto/*` packages, which is web-friendly (runs in-browser).
```

File: packages/common/README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
-# ATP Common Library
+# @atproto/common
 
-A library containing code which is shared between ATP packages.
+Shared Typescript code for other `@atproto/*` packages. This package is oriented towards writing servers, and is not designed to be browser-compatible.
```
Comment (devinivy) [inline packages/common/README.md:3]:
```suggestion
Shared TypeScript code for other `@atproto/*` packages. This package is oriented towards writing servers, and is not designed to be browser-compatible.
```

File: packages/crypto/README.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
-# ATP Crypto Library
+# @atproto/crypto
 
-ATP's common cryptographic operations.
+Typescript library providing basic cryptographic helpers as needed in [atproto](https://atproto.com).
```
Comment (devinivy) [inline packages/crypto/README.md:3]:
```suggestion
TypeScript library providing basic cryptographic helpers as needed in [atproto](https://atproto.com).
```

File: packages/identity/README.md [hunk 4]
```diff
@@ -1,3 +1,40 @@
-# ATP DID Resolver
+# @atproto/identity
 
-A library for resolving ATP's Decentralized ID methods.
+Typescript library for decentralized identities in [atproto](https://atproto.com) using DIDs and handles
```
Comment (devinivy) [inline packages/identity/README.md:3]:
```suggestion
TypeScript library for decentralized identities in [atproto](https://atproto.com) using DIDs and handles
```

File: packages/lex-cli/package.json [hunk 5]
```diff
@@ -1,6 +1,18 @@
 {
   "name": "@atproto/lex-cli",
   "version": "0.2.1",
+  "license": "MIT",
+  "description": "Typescript codegen tool for atproto Lexicon schemas",
```
Comment (devinivy) [inline packages/lex-cli/package.json:5]:
```suggestion
  "description": "TypeScript codegen tool for atproto Lexicon schemas",
```

File: packages/lexicon/README.md [hunk 6]
```diff
@@ -1,10 +1,9 @@
-# Lexicon
+# @atproto/lexicon: schema validation library
 
-Lexicon is the semantic schemas & contracts system for ATP. This library provides definitions and APIs for ATP software.
+Typescript implementation of the Lexicon data and API schema description language, which is part of [atproto](https://atproto.com).
```
Comment (devinivy) [inline packages/lexicon/README.md:3]:
```suggestion
TypeScript implementation of the Lexicon data and API schema description language, which is part of [atproto](https://atproto.com).
```

File: packages/pds/README.md [hunk 7]
```diff
@@ -1,3 +1,12 @@
-# ATP Personal Data Server (PDS)
+# @atproto/pds: Personal Data Server (PDS)
 
-The Personal Data Server (PDS). This is ATP's main server-side implementation.
+Typescript reference implementation of an atproto PDS.
```
Comment (devinivy) [inline packages/pds/README.md:3]:
```suggestion
TypeScript reference implementation of an atproto PDS.
```

File: packages/repo/README.md [hunk 8]
```diff
@@ -1,3 +1,12 @@
-# ATP Repo
+# @atproto/repo: Repository and MST
 
-The "ATP repository" core implementation (a Merkle Search Tree).
+Typescript library for atproto repositories, and in particular the Merkle Search Tree (MST) data structure.
```
Comment (devinivy) [inline packages/repo/README.md:3]:
```suggestion
TypeScript library for atproto repositories, and in particular the Merkle Search Tree (MST) data structure.
```

File: packages/xrpc-server/README.md [hunk 9]
```diff
@@ -1,4 +1,9 @@
-# XRPC Server API
+# @atproto/xrpc-server: atproto HTTP API server library
+
+Typescript library for implementing [atproto](https://atproto.com) HTTP API services, with Lexicon schema validation.
```
Comment (devinivy) [inline packages/xrpc-server/README.md:3]:
```suggestion
TypeScript library for implementing [atproto](https://atproto.com) HTTP API services, with Lexicon schema validation.
```

File: packages/xrpc/README.md [hunk 10]
```diff
@@ -1,4 +1,9 @@
-# XRPC API
+# @atproto/xrpc: atproto HTTP API Client
+
+Typescript client library for talking to [atproto](https://atproto.com) services, with Lexicon schema validation.
```
Comment (devinivy) [inline packages/xrpc/README.md:3]:
```suggestion
TypeScript client library for talking to [atproto](https://atproto.com) services, with Lexicon schema validation.
```

File: packages/aws/package.json [hunk 11]
```diff
@@ -1,17 +1,23 @@
 {
   "name": "@atproto/aws",
   "version": "0.1.1",
-  "main": "src/index.ts",
-  "publishConfig": {
-    "main": "dist/index.js",
-    "types": "dist/src/index.d.ts"
-  },
   "license": "MIT",
+  "description": "shared AWS cloud API helpers for atproto services",
```
Comment (estrattonbailey) [inline packages/aws/package.json:5]:
```suggestion
  "description": "Shared AWS cloud API helpers for atproto services",
```

File: packages/common-web/README.md [hunk 12]
```diff
@@ -1,3 +1,10 @@
...
+
+[![NPM](https://img.shields.io/npm/v/@atproto/common)](https://www.npmjs.com/package/@atproto/common)
```
Comment (estrattonbailey) [inline packages/common-web/README.md:5]:
```suggestion
[![NPM](https://img.shields.io/npm/v/@atproto/common)](https://www.npmjs.com/package/@atproto/common-web)
```

File: packages/common-web/package.json [hunk 13]
```diff
@@ -1,17 +1,22 @@
 {
   "name": "@atproto/common-web",
   "version": "0.2.0",
-  "main": "src/index.ts",
-  "publishConfig": {
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts"
-  },
   "license": "MIT",
+  "description": "shared web-platform-friendly code for atproto libraries",
```
Comment (estrattonbailey) [inline packages/common-web/package.json:5]:
```suggestion
  "description": "Shared web-platform-friendly code for atproto libraries",
```

File: packages/common/package.json [hunk 14]
```diff
@@ -1,17 +1,22 @@
 {
   "name": "@atproto/common",
   "version": "0.3.0",
-  "main": "src/index.ts",
-  "publishConfig": {
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts"
-  },
   "license": "MIT",
+  "description": "shared web-platform-friendly code for atproto libraries",
```
Comment (estrattonbailey) [inline packages/common/package.json:5]:
```suggestion
  "description": "Shared web-platform-friendly code for atproto libraries",
```

File: packages/syntax/README.md [hunk 15]
```diff
@@ -1,9 +1,16 @@
-# Syntax
+# @atproto/syntax: validation helpers for identifier strings
 
-Validation logic for AT identifiers - DIDs, Handles, NSIDs, and AT URIs.
+Validation logic for [atproto](https://atproto.com) identifiers - DIDs, Handles, NSIDs, and AT URIs.
+
+[![NPM](https://img.shields.io/npm/v/@atproto/crypto)](https://www.npmjs.com/package/@atproto/crypto)
```
Comment (estrattonbailey) [inline packages/syntax/README.md:5]:
```suggestion
[![NPM](https://img.shields.io/npm/v/@atproto/crypto)](https://www.npmjs.com/package/@atproto/syntax)
```

File: packages/syntax/package.json [hunk 16]
```diff
@@ -1,6 +1,20 @@
 {
   "name": "@atproto/syntax",
   "version": "0.1.1",
+  "license": "MIT",
+  "description": "validation for atproto identifiers and formats: DID, handle, NSID, AT URI, etc",
```
Comment (estrattonbailey) [inline packages/syntax/package.json:5]:
```suggestion
  "description": "Validation for atproto identifiers and formats: DID, handle, NSID, AT URI, etc",
```

**Review by estrattonbailey — COMMENTED** [review]
Small nits from me too :P

**Comment by bnewbold** [comment]
Ya'll got :eagle: :eyes:, thanks for catching all those little details!

**Pull request merged**